### PR TITLE
feat: upgrade react version

### PR DESCRIPTION
**Pull request by yigiterdev**
Related Issue: #22

### CODE REVIEW

File: src/modal/_pera-wallet-modal.scss [hunk 1]
```diff
@@ -1,6 +1,19 @@
 @import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");
 @import "../ui/styles/media-queries";
 
+* {
```
Comment (mucahit) [inline src/modal/_pera-wallet-modal.scss:4]: Let's move those under the `.pera-wallet-connect-modal`

File: src/modal/sign-toast/_pera-wallet-sign-txn-toast.scss [hunk 2]
```diff
@@ -1,5 +1,18 @@
 @import "../../ui/styles/media-queries";
 
+* {
```
Comment (mucahit) [inline src/modal/sign-toast/_pera-wallet-sign-txn-toast.scss:3]: Same here

**Review by mucahit — COMMENTED** [review]
thank you!! @yigiterdev

File: README.md [hunk 3]
```diff
@@ -28,6 +28,58 @@ Let's start with installing `@perawallet/connect`
 npm install --save @perawallet/connect
 ```
 
+<details>
+  <summary>Using with React 18</summary><br/>
+  
+  When you want to use `@perawallet/connect` library with React 18, you need to make some changes. After creating a new app with `npx create-react-app my-app`, the following changes should be made.
```
Comment (mucahit) [inline README.md:34]:
```suggestion
  When you want to use `@perawallet/connect` library with React 18, you need to make some changes. `react-scripts` stopped polyfilling some of the packages with the `react-scripts@5.x` version. After creating a new app with `npx create-react-app my-app` or in your react application, the following changes should be made.
```

File: README.md [hunk 4]
```diff
@@ -28,6 +28,58 @@ Let's start with installing `@perawallet/connect`
...
+
+1. Firstly, you need to install this packages.
```
Comment (mucahit) [inline README.md:36]:
```suggestion
1. Firstly, install the following packages.
```

File: README.md [hunk 5]
```diff
@@ -28,6 +28,58 @@ Let's start with installing `@perawallet/connect`
...
+
+```sh
+  npm install buffer,
+  npm install crypto-browserify
+  npm install process
+  npm install react-app-rewired
+  npm install stream-browserify
+```
+
+2. After that we need to override some webpack features. Create the following file in the root part of the project and copy the following code block into it.
```
Comment (mucahit) [inline README.md:46]:
```suggestion
2. After that you need to override some webpack features. Create the following file in the root directory of the project and copy the following code block into it.
```

File: README.md [hunk 6]
```diff
@@ -28,6 +28,58 @@ Let's start with installing `@perawallet/connect`
...
+    new webpack.ProvidePlugin({
+      process: "process/browser",
+      Buffer: ["buffer", "Buffer"]
+    })
+  ]);
+  return config;
+};
+```
+
+3. Finally, we need to change the npm scripts.
```
Comment (mucahit) [inline README.md:75]:
```suggestion
3. Finally, you need to update the npm scripts.
```

**Review by mucahit — COMMENTED** [review]
Thanks!

**Comment by fergusstrange** [comment]
Any progress on getting this one merged and released?

**Comment by mucahit** [comment]
> Any progress on getting this one merged and released?

Hey @fergusstrange, this will be released on Monday.

**Pull request merged**